Makefile

```diff
@@ -1,12 +1,16 @@
-INTEGRATION     := docker
-BINARY_NAME      = nri-$(INTEGRATION)
-SRC_DIR          = ./src/
-VALIDATE_DEPS    = golang.org/x/lint/golint
-INTEGRATIONS_DIR = /var/db/newrelic-infra/newrelic-integrations/
-CONFIG_DIR       = /etc/newrelic-infra/integrations.d
-GO_FILES        := ./src/
-GOOS             = GOOS=linux
-GO               = $(GOOS) go
+INTEGRATION     	:= docker
+BINARY_NAME      	= nri-$(INTEGRATION)
+SRC_DIR          	= ./src/
+VALIDATE_DEPS    	= golang.org/x/lint/golint
+INTEGRATIONS_DIR 	= /var/db/newrelic-infra/newrelic-integrations/
+CONFIG_DIR       	= /etc/newrelic-infra/integrations.d
+GO_FILES        	:= ./src/
+GOOS             	= GOOS=linux
+GO               	= $(GOOS) go
+GO_VERSION 			?= $(shell grep '^go ' go.mod | awk '{print $$2}')
+UBUNTU_VERSION 		?= "16.04"
+BUILDER_IMAGE_TAG 	?= "latest"
+BUILDER_IMAGE 		?= "ghcr.io/newrelic/coreint-automation:latest-go$(GO_VERSION)-ubuntu$(UBUNTU_VERSION)"
```

Comment:
Oh I think I misunderstood how we were using it, so we leverage pre-compiled images, we do not trigger the creation each time

Reply:
Yes, we only trigger when we want a new image built for a new version, OR if some changes are pushed to main.